### PR TITLE
[Impeller] make default glyph atlas size much bigger.

### PR DESCRIPTION
**Pull request by jonahwilliams**
Now that we no longer have a full CPU copy of the glyph atlas, it should be cheaper to make a larger initial atlas to reduce churn from resizing seen in applications like https://github.com/flutter/flutter/issues/138798

Requires https://github.com/flutter/engine/pull/52746

### CODE REVIEW

File: impeller/typographer/backends/skia/typographer_context_skia.cc [hunk 1]
```diff
@@ -194,16 +194,12 @@ static ISize OptimumAtlasSizeForFontGlyphPairs(
     const std::vector<FontGlyphPair>& pairs,
     std::vector<Rect>& glyph_positions,
     const std::shared_ptr<GlyphAtlasContext>& atlas_context,
-    GlyphAtlas::Type type,
     const ISize& max_texture_size) {
-  static constexpr auto kMinAtlasSize = 8u;
```
Comment (chinmaygarde) [inline impeller/typographer/backends/skia/typographer_context_skia.cc:199]: lol, thats comically small. I think this was from a time when the glyph atlas was per render pass.

**Review by chinmaygarde — APPROVED** [review]

**Comment by jonahwilliams** [comment]
The goldens on this are incredibly confusing.

**Comment by jonahwilliams** [comment]
I guess shrink the atlas size until I no longer see pink? Does that work lol

File: impeller/typographer/backends/skia/typographer_context_skia.cc (hunk 1 above)
Comment (jonahwilliams) [reply impeller/typographer/backends/skia/typographer_context_skia.cc:199]: not even a single emoji!

**Pull request merged**